back/src/test/kotlin/com/sos/smartopenspace/domain/SlotTest.kt

```diff
@@ -7,22 +7,28 @@ import java.time.LocalDate
 import java.time.LocalTime
 
 class SlotTest {
+  private val user = User("anemail@gmail.com", "Pepe")
   private val room1 = Room("1")
-  private val talk1 = Talk("talk1")
-  private val talk2 = Talk("talk2")
+  private val talk1 = Talk("talk1", speaker = user)
+  private val talk2 = Talk("talk2", speaker = user)
   private val aSlot = TalkSlot(LocalTime.parse("09:00"), LocalTime.parse("09:30"), LocalDate.now())
   private val otherSlot = TalkSlot(LocalTime.parse("09:30"), LocalTime.parse("09:45"), LocalDate.now())
   private val secondDaySlot = TalkSlot(LocalTime.parse("09:00"), LocalTime.parse("09:30"), LocalDate.now().plusDays(1))
 
-  private fun anyUser(talk: Talk) = User("augusto@sos.sos", "Augusto", "Augusto", mutableSetOf(), mutableSetOf(talk))
+  private fun anyUser(talk: Talk): User {
+    val user = User("augusto@sos.sos", "Augusto", "Augusto")
+    user.addTalk(talk)
+    return user
+  }
```

Comment:
Otra pavada:

```suggestion
  private fun anyUser(talk: Talk) = 
    User("augusto@sos.sos", "Augusto", "Augusto").apply {
        addTalk(talk)
    }
```

No se si conocés el `apply()`, como el `let` , el `run`, etc.